experiments/tf_trainer/tf_hub_classifier/model.py

```diff
@@ -54,7 +52,7 @@ def estimator(self, model_dir):
 
   def _model_fn(self, features, labels, mode, params, config):
     embedded_text_feature_column = hub.text_embedding_column(
-      key=self._text_feature_name, module_spec=FLAGS.model_spec)
+      key='text', module_spec=FLAGS.model_spec)
```

Comment:
Does this now want to be `base_model.TEXT_FEATURE_KEY` ? 
If not, maybe explain why?

Reply:
Done.